src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -127,6 +136,14 @@ function WorkspaceMemberDetailsPage({personalDetails, policy, route}: WorkspaceM
         Navigation.navigate(ROUTES.WORKSPACE_OWNER_CHANGE_CHECK.getRoute(policyID, accountID, 'amountOwed' as ValueOf<typeof CONST.POLICY.OWNERSHIP_ERRORS>));
     }, [accountID, policyID]);
 
+    // eslint-disable-next-line rulesdir/no-negated-variables
```

Comment:
Hmm I don't think you should use the lint rule here. 

You can just change the conditional

Reply:
Thanks